libkbfs/journal_server.go

```diff
@@ -29,6 +29,9 @@ type JournalServerStatus struct {
 	CurrentVerifyingKey kbfscrypto.VerifyingKey
 	JournalCount        int
 	UnflushedBytes      int64 // (signed because os.FileInfo.Size() is signed)
+	UnflushedPaths      []string
+
+	journalTlfIDs []TlfID
```

Comment:
Please have `Status()` just return a `[]TlfID`, and comment re. using it to fill in `UnflushedPaths` externally.